src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -107,6 +118,15 @@ public void start(final Map<String, String> parsedConfig) {
     // config as a side effect
     conn = SnowflakeConnectionServiceFactory.builder().setProperties(config).build();
 
+    // check if we are using snowpipe streaming ingestion
+    this.usesStreamingIngestion =
```

Comment:
this can be simplified with 
```suggestion
    this.usesStreamingIngestion = config != null
            && config.get(SnowflakeSinkConnectorConfig.INGESTION_METHOD_OPT) != null
            && config.get(SnowflakeSinkConnectorConfig.INGESTION_METHOD_OPT).
                .equalsIgnoreCase(IngestionMethodConfig.SNOWPIPE_STREAMING.toString());
```

defaults to false if nothing is set for ingestion method